manifests/params.pp

```diff
@@ -18,16 +17,44 @@
 
   case $::osfamily {
     'Debian': {
-      $libdir           = '/usr/share/jenkins'
-      $package_provider = 'dpkg'
+      $jenkins_user       = 'jenkins'
+      $jenkins_group      = 'jenkins'
+      $repo               = true
+      $homedir            = '/var/lib/jenkins'
+      $jenkinsdir         = $homedir
+      $default_plugin_dir = "${jenkinsdir}/plugins"
+      $libdir             = '/usr/share/jenkins'
+      $package_provider   = 'dpkg'
     }
     'RedHat': {
-      $libdir           = '/usr/lib/jenkins'
-      $package_provider = 'rpm'
+      $jenkins_user       = 'jenkins'
+      $jenkins_group      = 'jenkins'
+      $repo               = true
+      $homedir            = '/var/lib/jenkins'
+      $jenkinsdir         = $homedir
+      $default_plugin_dir = "${jenkinsdir}/plugins"
+      $libdir             = '/usr/lib/jenkins'
+      $package_provider   = 'rpm'
+    }
+    'OpenBSD': {
+      $jenkins_user       = '_jenkins'
+      $jenkins_group      = '_jenkins'
+      $repo               = false
+      $homedir            = '/var/jenkins'
+      $jenkinsdir         = "${homedir}/.jenkins"
```

Comment:
Repeating my earlier comment, why doesn't `$homedir == $jenkinsdir` as on all the other supported platforms?